src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,151 @@
 package core.basesyntax;
 
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Scanner;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static void main(String[] args) throws IOException {
+        Scanner scanner = new Scanner(System.in);
+        String command = scanner.next();
```

Comment:
краще зчитувати строку, а потім уже з нею здійснювати різні операції, наприклад поділити за пробілами і визначити відразу які аргументи ти передаєш для команди (щоб не робити це в cases)
```suggestion
        String command = scanner.nextLine();;
```